src/core/pages/PatientInfo.js

```diff
@@ -1,13 +1,21 @@
 import React from "react";
 import { Show, SimpleShowLayout, TextField, DateField } from "react-admin";
 
+import Breadcrumbs from "../common/Breadcrumbs";
+
+const breadcrumbsResource = [
+    { url: "/patients", title: "Patients", isActive: true },
+    { url: null, title: "Details", isActive: false }
+];
+
 /**
  * This component returns content of patient information page
  *
  * @author Bogdan Shcherban <bsc@piogroup.net>
  */
-const PatientInfo = props => {
-    return (
+const PatientInfo = props => (
+    <div>
```

Comment:
to avoid clutttering the dom, prefer adding a react `Fragment` rather than a div.

https://reactjs.org/docs/fragments.html

Reply:
Changed